test_src/sljitTest.c

```diff
@@ -6318,29 +6319,27 @@ static void test64(void)
 		printf("Run test64\n");
 
 	/* lock next allocation; see sljit_test_malloc_exec() */
-	test64_ptr_hack = (void *)64;
-	malloc_addr = (sljit_uw)SLJIT_MALLOC_EXEC(1024);
+	malloc_addr = (sljit_uw)SLJIT_MALLOC_EXEC(4096, NULL);
```

Comment:
the generated code used to fit in 130 or so bytes, using 1024 (as done originally) should allow for it to be allocated that exactly and probably remove the need for the max_offset?